src/sidebar/util/thread-annotations.js

```diff
@@ -0,0 +1,75 @@
+import buildThread from './build-thread';
+import memoize from './memoize';
+import * as metadata from './annotation-metadata';
+import { generateFacetedFilter } from './search-filter';
+import filterAnnotations from './view-filter';
+import { shouldShowInTab } from './tabs';
+
+/** @typedef {import('./build-thread').Thread} Thread */
+/** @typedef {import('./build-thread').Options} BuildThreadOptions */
+/** @typedef {import('../store/modules/selection').ThreadState} ThreadState */
+
+// Sort functions keyed on sort option
+const sortFns = {
+  Newest: function (a, b) {
```

Comment:
Minor suggestion - `sortFns` could be streamlined a little using modern syntax (eg. `Newest: (a, b) => a.updated > b.updated`)

Reply:
I'll keep this in my back pocket as I'm sure I'm going to be in this module a lot in the next few days.